### PR TITLE
code review

### PR DESCRIPTION
**Pull request by shubham-singh**

### CODE REVIEW

File: models/book.model.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  author: String,
+  price: Number,
+  img: [],
+  description: String,
+  category: String,
+  inStock: Boolean
+});
+
+// const Book = mongoose.model('book', BookSchema);
```
Comment (barbaricankit) [inline models/book.model.js]: remove comments

File: models/book.model.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+const BookSchema = new Schema({
+  title: {
+    type: String,
+    required: [true, 'Title is required']
+  },
+  author: String,
+  price: Number,
+  img: [],
```
Comment (barbaricankit) [inline models/book.model.js]: why img as array?

**Review by barbaricankit — COMMENTED** [review]
Very clean code. It was really nice reviewing your code

File: models/book.model.js (hunk 2 above)
Comment (shubham-singh) [reply models/book.model.js]: i might add more images of same product that's why, actually i was thinking of making it an object with keys like default, front and back etc etc

File: models/book.model.js (hunk 1 above)
Comment (shubham-singh) [reply models/book.model.js:18]: yeah i will remove them, was just testing different with different documents

File: db/db.connect.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const username = process.env['username'];
+const password = process.env['password'];
+
+// const dbURI = `mongodb+srv://${username}:${password}@neog-cluster.wiph4.mongodb.net/books`;
```
Comment (duhbhavesh) [inline db/db.connect.js]: comment can be removed

File: index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+const mongoose = require('mongoose');
+const dbConnect = require('./db/db.connect.js');
+const cors = require('cors');
+const bookRoute = require('./routes/book.route');
+const Book = require('./models/book.model.js');
+
+const app = express();
+
+
+// Connecting to databse
```
Comment (duhbhavesh) [inline index.js]: Here as well

**Review by duhbhavesh — COMMENTED** [review]
Hi @shubham-singh 

Overall everything's good, I've suggested only minor change.

**Comment by duhbhavesh** [comment]
Looks good👍

**Comment by barbaricankit** [comment]
Looks Good

**Pull request merged**